package.json

```diff
@@ -40,6 +40,7 @@
     "babel": "^5.8.25",
     "babel-core": "^5.8.25",
     "babel-eslint": "^4.1.1",
+    "babel-relay-plugin": "0.7.1",
```

Comment:
I updates the path in https://github.com/facebook/relay/blob/master/scripts/jest/preprocessor.js#L66 to https://github.com/facebook/relay/blob/master/scripts/babel-relay-plugin/package.json and that made all the tests pass :)